applications/IgaApplication/custom_processes/assign_iga_external_conditions_process.h

```diff
@@ -0,0 +1,158 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:   Andrea Gorgi, Polytimi Zisimoupoulu
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+
+#include "processes/process.h"
+
+#include "utilities/function_parser_utility.h"
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+/* @class AssignIgaExternalConditionsProcess
+ * @ingroup IgaApplication
+ **/
+class KRATOS_API(IGA_APPLICATION) AssignIgaExternalConditionsProcess
+    : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    typedef std::size_t IndexType;
+    typedef std::size_t SizeType;
+
+
+
+    /// Pointer definition of AssignIgaExternalConditionsProcess
+    KRATOS_CLASS_POINTER_DEFINITION(AssignIgaExternalConditionsProcess);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Constructor
+    AssignIgaExternalConditionsProcess(
+        Model& rModel,
+        Parameters ThisParameters);
+
+    /// Destructor.
+    ~AssignIgaExternalConditionsProcess() = default;
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    void Execute() override;
+
+    void ExecuteInitializeSolutionStep() override {
+        Execute();
+    };
+
+    const Parameters GetDefaultParameters() const override
+    {
+        const Parameters default_parameters = Parameters(R"(
+        {
+          "echo_level": 0,
+          "analysis_model_part_name": "IgaModelPart",
```

Comment:
```suggestion
          "model_part_name": "IgaModelPart",
```
I don't see the need to change the customary input name we use.